test_runner/regress/test_disk_usage_eviction.py

```diff
@@ -668,11 +655,11 @@ def test_fast_growing_tenant(neon_env_builder: NeonEnvBuilder, pg_bin: PgBin, or
         finish_tenant_creation(env, tenant_id, timeline_id, min_expected_layers)
 
     tenant_layers = count_layers_per_tenant(env.pageserver, map(lambda x: x[0], timelines))
-    (total_on_disk, _, _) = poor_mans_du(env, map(lambda x: x[0], timelines), env.pageserver, False)
+    (total_on_disk, _, _) = poor_mans_du(env, map(lambda x: x[0], timelines), env.pageserver, True)
 
     # cut 10 percent
```

Comment:
nit: comment is stale now, although i'm not sure why this change was required

Reply:
the false -> true logs the layers which we ended up with before assertions, leaving something useful to the log about what was the overall state before the assertion failed.

good spotting with the obsolete comment.